linode/manager
# Add Cypress E2E tests for DBaaS cluster creation

## Description

**What does this PR do?**
* Adds Cypress tests for DBaaS cluster creation using mocked data and API responses
* Invalidates database list query upon database finished or failed event

## How to test

**What are the steps to reproduce the issue or verify the changes?**
* To verify E2E tests, run the following from the repo root and confirm that tests succeed:
  * `yarn cy:run -s '**/create-database.spec.ts'`
* To verify React Query changes, navigate to DBaaS landing page, create a DB cluster, and confirm that Manager automatically and immediately makes a request to `/databases/instances` after receiving an event with action `database_create` and status `finished`
* Perform additional manual testing to ensure that React Query changes do not cause any unintended side effects
